js/render/draw_extrusion.js

```diff
@@ -159,7 +159,8 @@ function drawExtrusion(painter, source, layer, coord) {
     programConfiguration.setUniforms(gl, program, layer, {zoom: painter.transform.zoom});
 
     if (image) {
-        setPattern(image, tile, coord, painter, program, true);
+        setPattern(image, painter, program);
+        setPattern.setTile(tile, painter, program, true);
```

Comment:
This is a tiny detail, but can `set_pattern.js` export both functions as part of an object instead of making `setTile` a property on `setPattern`?